src/ConfigProvider.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\I18n\PhoneNumber;
+
+use Laminas\ServiceManager\ConfigInterface;
+
+/**
+ * @see ConfigInterface
+ *
+ * @psalm-import-type ServiceManagerConfigurationType from ConfigInterface
+ *
+ * @psalm-type PackageConfig = array{
+ *     default-country-code?: non-empty-string|null
+ * }
+ */
+final class ConfigProvider
```

Comment:
We could drop a line here, explaining that this is for usage in some sort of config provider?